tagstudio/tests/test_library.py

```diff
@@ -194,8 +190,8 @@ def test_add_field_tag(library, entry_full, generate_tag):
     library.add_field_tag(entry_full, tag, tag_field.type_key)
 
     # Then
-    results = library.search_library(FilterState(id=entry_full.id))
-    tag_field = results[0].tag_box_fields[0]
+    results = library.get_entry_full(entry_full.id)
+    tag_field = results.tag_box_fields[0]
```

Comment:
Nit: match singular result name from other `get_entry_full()` calls for consistency
```suggestion
    result = library.get_entry_full(entry_full.id)
    tag_field = result.tag_box_fields[0]
```